frontend/javascripts/oxalis/controller/viewmodes/plane_controller.js

```diff
@@ -134,21 +134,18 @@ class PlaneController extends React.PureComponent<Props> {
     // catchers (only necessary for HammerJS). We should refactor the
     // InputMouse handling so that this is not necessary anymore.
     // See: https://github.com/scalableminds/webknossos/issues/3475
-    const tdId = `inputcatcher_${OrthoViews.TDView}`;
-    Utils.waitForElementWithId(tdId).then(() => {
-      OrthoViewValuesWithoutTDView.forEach(id => {
-        const inputcatcherId = `inputcatcher_${OrthoViews[id]}`;
-        Utils.waitForElementWithId(inputcatcherId).then(el => {
-          if (!document.body.contains(el)) {
-            console.error("el is not attached anymore");
-          }
-          this.input.mouseControllers[id] = new InputMouse(
-            inputcatcherId,
-            this.getPlaneMouseControls(id),
-            id,
-            true,
-          );
-        });
+    OrthoViewValuesWithoutTDView.forEach(id => {
```

Comment:
Waiting for the 3d viewport did not succeed here, if it's hidden initially. In that case, the mouse handlers were not set up. Since, there is a `waitForElementWithId` two lines below for **each** viewport, I'd hope that this change doesn't break anything.

Reply:
I think this won't be a problem, as the `waitForElementWithId` works with promises. The onliest thing I see is that if one of the viewports is never rendered, that there will be endless calls of `window.requestAnimationFrame` as long as wk is opened in a browser tab. 
I think this is not optimal. Maybe we can add a timeout to this function and retrigger adding the mouse controllers for all inputcatchers that were newly rendered when the layout is changed.
But this would be a thing for another PR.

Note:
```
export function waitForElementWithId(elementId: string): Promise<*> {
  const tryToResolve = resolve => {
    const el = document.getElementById(elementId);
    if (el) {
      resolve(el);
    } else {
      window.requestAnimationFrame(() => tryToResolve(resolve));
    }
  };
  return new Promise(tryToResolve);
}
```

Reply:
Good point! I cross-referenced this comment in the corresponding issue: https://github.com/scalableminds/webknossos/issues/3475

I agree that this is out-of-scope for this PR.